pkg/util/kubernetes/clustername/clustername.go

```diff
@@ -62,18 +62,21 @@ func init() {
 	}
 }
 
-func getClusterName(data *clusterNameData) string {
+func getClusterName(data *clusterNameData, hostname string) string {
 	data.mutex.Lock()
 	defer data.mutex.Unlock()
 
 	if !data.initDone {
 		data.clusterName = config.Datadog.GetString("cluster_name")
 		if data.clusterName != "" {
 			log.Infof("Got cluster name %s from config", data.clusterName)
-			if !validClusterName.MatchString(data.clusterName) || len(data.clusterName) > 40 {
-				log.Errorf("%q isn’t a valid cluster name. It must be dot-separated tokens where tokens "+
-					"start with a lowercase letter followed by up to 39 lowercase letters, numbers, or "+
-					"hyphens, and cannot end with a hyphen nor have a dot adjacent to a hyphen.", data.clusterName)
+			// the host alias "clusterName-hostName" must not exceed 255 chars
+			hostAlias := data.clusterName + "-" + hostname
```

Comment:
in the PR description you say:
>  introduce a 255 chars limitation on "hostname-clustername"
But here you do the opposite: "clustername-hostname"

Reply:
the right one is `(nodeName + "-" + clusterName)` fixing this